packages/forms/resources/views/components/repeater/index.blade.php

```diff
@@ -79,7 +82,16 @@ class="items-start gap-4"
                                 $extraItemActions,
                                 fn (Action $action): bool => $action(['item' => $uuid])->isVisible(),
                             );
-                            $itemHasToolbar = $isReorderableWithDragAndDrop || $isReorderableWithButtons || filled($itemLabel) || $isCloneable || $isDeletable || $isCollapsible || $visibleExtraItemActions;
+                            $cloneAction = $cloneAction(['item' => $uuid]);
```

Comment:
I don't think you need to reassign the var, invoking it modifies the existing object

Reply:
I just tried and it doesn't, so will need to stay as is.